apps/core/graphql/types/user.ex

```diff
@@ -0,0 +1,74 @@
+defmodule Core.GraphQL.Types.User do
+
+  use Absinthe.Schema.Notation
+  use Absinthe.Relay.Schema.Notation
+  alias Absinthe.Relay.Connection
+  alias Core.{GraphQL.Schema, User, Repo}
+  import Ecto.Query, only: [from: 2]
+  import Canada, only: [can?: 2]
+
+  #
+  # Mutations
+  #
+  object :user_mutations do
+    @desc """
+    Add/Edit a User to DB if success
+    """
+    payload field :user do
+      input do
+        field :id, non_null(:id)
```

Comment:
**id** is not required to add user. I am using the `id` to be able to create or edit a user. Now, as you can see `id` is not null, however at the time of creating the user is added normally and does not arise any type of error.